tests/test_cli.py

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `mlbench_core.cli` package."""
+
+import pytest
+from mlbench_core.cli import cli_group
+from google.auth.exceptions import DefaultCredentialsError
+
+# import click
+
+create_gcloud = cli_group.commands["create-cluster"].commands["gcloud"].main
+delete_gcloud = cli_group.commands["delete-cluster"].commands["gcloud"].main
+get_status = cli_group.commands["status"]
+delete_run = cli_group.commands["delete"]
+
+CREATE_GCLOUD_DEFAULTS = {
+    "machine_type": "n1-standard-4",
+    "disk_size": 50,
+    "num_cpus": 1,
+    "num_gpus": 0,
+    "gpu_type": "nvidia-tesla-k80",
+    "zone": "europe-west1-b",
+    "preemptible": False,
+}
+
+
+def get_gcloud_cmd_line(args, option_dict=None):
+    """
```

Comment:
Please follow google docs style guidelines: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

So
```
"""Short description

(optional long description)

Args:
    arg1 (type): Description

Returns:
    type: Description
"""
```

Fixtures should at least have the short description to describe what they do, tests should also have a short description saying what they're testing for.